src/components/hero.component.ts

```diff
@@ -8,7 +8,7 @@ import HeroImg from '@assets/images/hero.webp';
   standalone: true,
   template: `
     <article
-      class="hero h-dvh min-h-[600px] p-4 md:p-8 bg-cover bg-center bg-no-repeat relative after:content-[''] after:absolute after:inset-0 after:z-[1] after:bg-gradient-to-b after:from-[rgba(107,9,141,0.8)] after:via-[rgba(0,31,33,0.6)] after:to-[rgba(3,24,57,0.7)]"
+      class="h-dvh min-h-[600px] p-4 md:p-8 bg-cover bg-center bg-no-repeat relative after:content-[''] after:absolute after:inset-0 after:z-[1] after:bg-gradient-to-b after:from-[rgba(107,9,141,0.8)] after:via-[rgba(0,31,33,0.6)] after:to-[rgba(3,24,57,0.7)]"
```

Comment:
me parece que es muy complejo este código, podemos usar @apply en tailwind para encapsular parte de tailwind en una clase de css . Aqui dejo una referencia: https://tailwindcss.com/docs/functions-and-directives#apply-directive